### ISSUE TITLE
working directory

### ISSUE THREAD

**Issue by ayushkovskiy** [neuromation/platform-client-python#1235]

**Comment by dalazx** [comment]
@ayushkovskiy in order to achieve this, we need to patch both `platform-api` and `platform-monitoring` to cover `neuro run` and `neuro save` correspondingly.

**Comment by asvetlov** [comment]
`neuro run` executes a Docker image.
The image is built from Dockerfile.
The Dockerfile has a `WORKDIR` directive to specify the current directory.
No need to add new functionality to `neuro run` but prepare images properly instead.

Did I miss something?

**Comment by ayushkovskiy** [comment]
I agree that the issue is not for `neuro run`, but rather for `neuro save`.
Let's extend (in future) `neuro save` on the side of both `platform-monitoring` and `neuro` so that it accepts parameter `-c/--change` (see https://docs.docker.com/engine/reference/commandline/container_commit/). Thus, we could tweak `Makefile`'s `make setup` instruction to override `WORKDIR` to the project's directory during `neuro save`.
